app/src/main/java/ca/chronofit/chrono/SplashActivity.kt

```diff
@@ -64,4 +62,8 @@ class SplashActivity : BaseActivity() {
             }, SPLASH_TIMEOUT
         )
     }
+
+    companion object {
+        private const val SPLASH_TIMEOUT = 3000L // Timeout delay
```

Comment:
i always thought of companion objects as being equivalent to javas "public static" variables. Whats the point of making their visibility private here?

Reply:
I think this should be a top-level declaration. Or just leave it the way it was before. 
https://stackoverflow.com/questions/49969319/kotlin-difference-between-constant-in-companion-object-and-top-level